Mathlib/RingTheory/Adjoin/Field.lean

```diff
@@ -97,6 +97,16 @@ theorem IsIntegral.mem_range_algebraMap_of_minpoly_splits [Algebra K L] [IsScala
     x ∈ (algebraMap K L).range :=
   int.mem_range_algHom_of_minpoly_splits h (IsScalarTower.toAlgHom R K L)
 
+theorem minpoly_split_add_algebraMap [Algebra K L] {x : L} (r : K)
+    (g : (minpoly K x).Splits (algebraMap K L)) :
+    (minpoly K (x + algebraMap K L r)).Splits (algebraMap K L) := by
+  simpa [minpoly.add_algebraMap] using g.comp_X_sub_C r
+
+theorem minpoly_split_sub_algebraMap [Algebra K L] {x : L} (r : K)
```

Comment:
`split` → `splits`, and the order doesn't make sense to me. Probably `splits_minpoly_{add,sub}_algebraMap`

Do you also want to add the `algebraMap K L r +/- x` variants?

Reply:
Thank you! I'll rename it to `minpoly_{add, sub}_algebraMap_splits` which is the order of the first four terms that can be read in the conclusion `(minpoly K (x + algebraMap K L r)).Splits (algebraMap K L)`.